src/main/java/io/r2dbc/proxy/core/Binding.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2018 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.r2dbc.proxy.core;
+
+/**
+ * Represent an operation of {@link io.r2dbc.spi.Statement#bind}.
+ *
+ * @author Tadaya Tsuyukubo
+ * @see Bindings.IndexBinding
+ * @see Bindings.IdentifierBinding
+ */
+public interface Binding {
```

Comment:
Binding keeps reappearing across various implementations. That's just something I noticed, don't want to fix that right now.